docs/blog/write-the-docs-2022-stats.rst

```diff
@@ -0,0 +1,88 @@
+.. post:: Jan 13, 2023
+   :tags: stats, year in review
+   :author: Eric Holscher
+
+Write the Docs 2022 Stats
+=========================
+
+This year we have done a :doc:`/blog/2023-january-community-update` post along with our stats.
+Check there for context around the community going into 2023.
+
+Unfortunately our Podcast has been in hiatus for a while with no new episodes in 2022,
+so we have removed it from the stats post.
+
+2022 Stats
+----------
+
+We have shared our high-level community stats for the past 5 years,
+and will continue to do it each year going forward.
+
+You can read our previous posts from 2016_, 2017_, 2018_, 2019_, 2020_, 2022_.
+
+.. _2022: https://www.writethedocs.org/blog/write-the-docs-2022-stats/
+.. _2020: https://www.writethedocs.org/blog/write-the-docs-2020-stats/
+.. _2019: https://www.writethedocs.org/blog/write-the-docs-2019-stats/
+.. _2018: https://www.writethedocs.org/blog/write-the-docs-2018-stats/
+.. _2017: https://www.writethedocs.org/blog/write-the-docs-2017-stats/
+.. _2016: https://www.writethedocs.org/blog/write-the-docs-2016-year-in-review/
+
+Conferences
+~~~~~~~~~~~
+
+* 650 attendees in Portland (down from 650)
+* 225 attendees in Prague (up from 330)
+* 100 attendees in Australia (no conference in 2022)
+
+We were excited to see continued success of the virtual conferences,
+but it definitely felt like more people are slowly yearning to return to in-person.
+The world has also had another large shock with the war in Ukraine,
+which we expect to have lowered our 2022 attendance in Prague.
+
+Slack Network
+~~~~~~~~~~~~~
+
+* 18,339 :doc:`members </slack>` (up from 15,012)
+
+Newsletters
+~~~~~~~~~~~
+
+* 10,558 :doc:`subscribers </newsletter>` (up from 9,237)
+
+This is across all our various mailing lists (conference-specific and community newsletter).
+
+Meetups
+~~~~~~~
+
+* Over 10,000 :doc:`members </meetups/index> (too many to count)
+
+Last year we stopped counting total meetups because we launched Quorum meetups that cover a larger geographic area.
+We're excited to see a return to in-person meetups,
+with the easing of COVID concerns in various areas where we have meetups.
+
+Website
+~~~~~~~
+
+* 270,000 sessions (up from 250,000)
+* 440,000 page views (up from 395,000)
+
+Our website traffic is generally pretty stable,
+but we do see spikes around our conferences.
+This number comes from Google Analytics which many users are blocking,
```

Comment:
Should we look into changing to a more neutral analytics tool? @StarfallProjects has recommended on in the past I believe.

Reply:
Yea, we could use Plausible. It costs like $10/mo, but I think we can afford it :)

Reply:
Tracking in https://github.com/writethedocs/www/issues/1871